src/ClientOperations/AbstractSubscriptionOperation.php

```diff
@@ -337,6 +337,7 @@ protected function eventAppeared(ResolvedEvent $e): void
         }
 
         if ($this->verboseLogging) {
+            /** @psalm-suppress PossiblyNullReference */
```

Comment:
What is possibly null according to psalm here? And why are you sure it won't be null?

Reply:
`$e->originalEvent` could be null, but an event appeared, so it's not.